lib/main.dart

```diff
@@ -94,53 +94,69 @@ class _FoodFraudState extends State<FoodFraudListPage> {
 
   Widget _buildItem(BuildContext context, DataSnapshot snapshot,
       Animation<double> animation, int index) {
+
```

Comment:
Please use variable names in english. It would be good to create a class to hold this data and use a method to get instance of the class from the snapshot.